src/visor/config.go

```diff
@@ -81,23 +81,23 @@ func (c Config) Verify() error {
 	}
 
 	if c.UnconfirmedVerifyTxn.BurnFactor < params.UserVerifyTxn.BurnFactor {
-		return fmt.Errorf("UnconfirmedVerifyTxn.BurnFactor must be >= params.UserVerifyTxn.BurnFactor (%d)", params.UserVerifyTxn.BurnFactor)
+		return fmt.Errorf("unconfirmedVerifyTxn.BurnFactor must be >= params.UserVerifyTxn.BurnFactor (%d)", params.UserVerifyTxn.BurnFactor)
```

Comment:
Undo the capitalization changes of the intro params in log messages

Reply:
Done